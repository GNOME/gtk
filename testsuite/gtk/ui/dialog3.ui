<!-- invalid dialog markup attribute -->
<interface>
  <object class="GtkDialog">
    <child type="action">
      <object class="GtkButton" id="button_ok">
      </object>
    </child>
    <action-widgets>
      <action-widget response="warbl">button_ok</action-widget>
    </action-widgets>
  </object>
</interface>
