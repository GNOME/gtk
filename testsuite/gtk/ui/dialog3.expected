ERROR: gtk-builder-error-quark 6
.:9:40 Could not parse enum: 'warbl'
